### PR TITLE
Modal register login

### PR DESCRIPTION
**Pull request by wafaworks**
Finished the login/register modal and changed connexion button format

### CODE REVIEW

File: app/Resources/views/modules/header.html.twig [hunk 1]
```diff
@@ -37,7 +39,7 @@
                         </ul>
                     </li>
                     {% else %}
-                    <li class="navbar-menu"><a class="navbar-link" href="{{ path('member_connection') }}">{{ 'connection' |trans |upper}}</a></li>
+                    <button id="connexionButton" type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#registerLoginModal">{{ 'connection' |trans |upper}}</button>
```
Comment (ngdo-pro) [inline app/Resources/views/modules/header.html.twig:42]: I think the style on this button seems pretty cheap, the borders are not great, we should remove them.
In order to do so, first remove the btn-info class here

File: web/dev/sass/views/_header.sass [hunk 2]
```diff
@@ -57,6 +57,18 @@
     height: 6px
     width: 32px
 
+#connexionButton
+  background-color: transparent
+  border: solid 2px $font-color3
```
Comment (ngdo-pro) [inline web/dev/sass/views/_header.sass]: second, remove the border here
thrid, add the color: $font-color1

**Review by ngdo-pro — CHANGES_REQUESTED** [review]
Just a style to change in my opinion, excellent work !

File: web/dev/sass/views/_modalRegisterLogin.sass [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+#fosLoginForm
+  margin-top: 7rem
+  label
+    margin-bottom: 4rem
+    width: 14rem
+    text-align: right
+    margin-right: 1rem
+    margin-left: -10rem
```
Comment (Kaji-U) [inline web/dev/sass/views/_modalRegisterLogin.sass]: On your label you have 3 parameters for the margin option. Make just one like "margin: 0 1rem 4rem -10rem"

File: web/dev/sass/views/_modalRegisterLogin.sass [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+.fos_user_registration_register
+  margin-top: 6rem
+  label
+    margin-bottom: 4rem
+    width: 20rem
+    margin-left: -5rem
+    margin-right: 2rem
+    text-align: right
+    font-style: normal
```
Comment (Kaji-U) [inline web/dev/sass/views/_modalRegisterLogin.sass:42]: Same here

**Review by Kaji-U — DISMISSED** [review]

File: app/Resources/views/modules/header.html.twig (hunk 1 above)
Comment (shog189) [reply app/Resources/views/modules/header.html.twig:42]: I agree

File: web/dev/sass/views/_header.sass (hunk 2 above)
Comment (wafaworks) [reply web/dev/sass/views/_header.sass]: changed it to white

**Review by ngdo-pro — APPROVED** [review]

File: web/dev/sass/views/_modalRegisterLogin.sass (hunk 3 above)
Comment (wafaworks) [reply web/dev/sass/views/_modalRegisterLogin.sass]: done

File: web/dev/sass/views/_modalRegisterLogin.sass (hunk 4 above)
Comment (wafaworks) [reply web/dev/sass/views/_modalRegisterLogin.sass:42]: same

**Pull request merged**